README.md

```diff
@@ -27,6 +27,7 @@ ShipFosdick.configure do |config|
   config.client_name = ENV.fetch('CLIENT_NAME')
   config.client_code = ENV.fetch('CLIENT_CODE')
   config.adcode = ENV.fetch('ADCODE')
+  config.shipment_updater_class = "ShipFosdick::IndividualShipmentUpdater"
```

Comment:
Out of curiosity, I've made something similar for QL, but I went with a callable object (class, block, w/e). Here my question is, what's the tradeoff between referencing the class by it's constant and the string I see above?